### PR TITLE
Ticket 40 : fix some bugs

### PR DESCRIPTION
**Pull request by maximFeldman1**
The table is full width
Button of dropdown menu without background (transparent)
Brand  product is dropdown (you can use from the list)
Submit is disable until the fields are not filled
The header is located only on the main page
You can go back from create page to main page without validation

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -18,3 +18,7 @@
   display: flex;
   justify-content: center;
 }
+
+.p-dropdown {
+  min-width: 13em;
```
Comment (itairom-appsforce) [inline src/index.css:23]: use pixels? @amitmarko

File: src/ui-kit/modal/EditModal/index.tsx [hunk 2]
```diff
@@ -35,13 +42,15 @@ export const EditModal = ({
...
-    <Dialog visible={visible} onHide={onCancel} header="Edit Product">
-      <ProductForm
-        initialValues={productData}
-        onCancel={onCancel}
-        onDone={onDone}
-        onSubmit={onSubmit}
-      />
-    </Dialog>
+    <Root>
+      <Dialog visible={visible} onHide={onCancel} header="Edit Product">
```
Comment (itairom-appsforce) [inline src/ui-kit/modal/EditModal/index.tsx:46]: use i18 for header

Comment (maximFeldman1) [reply src/ui-kit/modal/EditModal/index.tsx:46]: Fixed

File: src/index.css (hunk 1 above)
Comment (maximFeldman1) [reply src/index.css:23]: Fixed

**Pull request merged**